eftcamb/06_abstract_EFTCAMB_model.f90

```diff
@@ -78,6 +78,7 @@ procedure :: compute_rhoQPQ              => EFTCAMBModelComputeRhoQPQ
         procedure :: compute_Einstein_factors    => EFTCAMBModelComputeEinsteinFactors                    !< subroutine that computes the Einstein equations factors. For details refer to the numerical notes.
         procedure :: compute_pi_factors          => EFTCAMBModelComputePiFactors                          !< subroutine that computes the pi field equations factors. For details refer to the numerical notes.
         procedure :: compute_tensor_factors      => EFTCAMBModelComputeTensorFactors                      !< subroutine that computes the factors for the tensor propagation equation. For details refer to the numerical notes.
```

Comment:
Sorry if this sounds really pedantic :)
can you add a space between the name and the arrow and maintain alignment of the different lines?